library/src/tensor_operation_instance/gpu/grouped_conv3d_fwd_scale/xdl/device_grouped_conv3d_fwd_xdl_scale_ndhwgc_gkzyxc_ndhwgk_bf16_instance.cpp

```diff
@@ -21,7 +21,7 @@ void add_device_grouped_conv3d_fwd_xdl_scale_ndhwgc_gkzyxc_ndhwgk_bf16_instances
                                                                 BF16,
```

Comment:
Please remove changes from these files and create:
device_grouped_conv3d_fwd_xdl_dynamic_ndhwgc_gkzyxc_ndhwgk_bf16_instance.cpp files for int8, fp16, bf16, fp32, for 2d and 3d